bundles/org.openhab.binding.mqtt.homeassistant/src/main/java/org/openhab/binding/mqtt/homeassistant/internal/ComponentSensor.java

```diff
@@ -61,8 +65,12 @@ public ComponentSensor(CFactory.ComponentConfiguration componentConfiguration) {
             value = new TextValue();
         }
 
+        String objectId = componentConfiguration.getHaID().objectID;
+
+        boolean trigger = triggerObjectIDs.contains(objectId);
```

Comment:
This is the part I am not happy with.
The check against 2 explicit values is not future save. I see many people wanting to have different values in the list also to make their device work.

Reply:
Oh, sorry, I think that I finally understood your concern regarding the 'action' and 'click' objectIDs topic :facepalm:  I was thinking all the time that those were universal HA objects used to send event information, but know I see that they are Zigbee2Mqtt specific. I'm so sorry for the confusion :innocent:

Ok, would you accept changing the logic to detect if the icon name starts with "mdi:gesture"? That is one of your proposals in #6878 and seems to fit fine with Zigbee2Mqtt, so I would be happy to implement it :+1:

Reply:
@jochen314 Can you comment on this?